### PR TITLE
Ignore valid links

### PR DESCRIPTION
**Pull request by core-man**
Addresses #57. Merge this PR after https://github.com/seismo-learn/seismology101/pull/350.

### CODE REVIEW

File: .github/workflows/check-links.yml [hunk 1]
```diff
@@ -35,9 +35,11 @@ jobs:
           --exclude "https://fonts.gstatic.com"
           --exclude "https://pkgs.org"
           --exclude "https://www.fdsn.org/xml/station"
+          --exclude "https://www.fdsn.org/webservices/"
```
Comment (seisman) [inline .github/workflows/check-links.yml:38]: Ignore all URLs starting with "https://www.fdsn.org/"
```suggestion
          --exclude "https://www.fdsn.org/"
```

Comment (core-man) [reply .github/workflows/check-links.yml:38]: But sometimes, the main URL may be the same, but the sub-url changes.

Comment (seisman) [reply .github/workflows/check-links.yml:38]: The error message is:
```
error trying to connect: error:1416F086:SSL routines:tls_process_server_certificate:certificate verify failed:../ssl/statem/statem_clnt.c:1915: (unable to get local issuer certificate))
```
So, something wrong with their SSL certification. I believe all their URLs are "invalid".

File: .github/workflows/check-links.yml [hunk 2]
```diff
@@ -3,7 +3,7 @@ name: Check Links
 
 on:
   # Uncomment the 'pull_request' line below to trigger the workflow in PR
-  # pull_request:
+  pull_request:
```
Comment (seisman) [inline .github/workflows/check-links.yml:6]:
```suggestion
  # pull_request:
```

**Review by seisman — APPROVED** [review]

**Pull request merged**